ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -397,14 +397,41 @@ impl fmt::Display for Module {
             )?;
         }
         write!(f, "\nentry:")?;
-        for (i, inst) in self.insts().iter_enumerated() {
-            write!(f, "\n    {}", inst.display(i, self))?;
+        let mut inst_iter = self.iter_inst_idxs();
+        while let Some(inst_i) = inst_iter.next(self) {
+            write!(f, "\n    {}", self.insts[inst_i].display(inst_i, self))?;
         }
 
         Ok(())
     }
 }
 
+/// An iterator over instruction indices. This skips "forwarding" instructions: in other words, it
```

Comment:
"forwarding"/"proxy"?

Reply:
Fixed in 1a0d2290e3f31999c68a1ffe3d7fdf591dadd98a.